app/cdap/components/PipelineTriggers/EnabledTriggersTab/EnabledCompositeTriggerRow.tsx

```diff
@@ -20,78 +20,78 @@ import { connect } from 'react-redux';
 import EnabledInlineTriggerRow from 'components/PipelineTriggers/EnabledTriggersTab/EnabledInlineTriggerRow';
 import {
   disableSchedule,
-  getGroupPipelineInfo,
+  getCompositePipelineInfo,
 } from 'components/PipelineTriggers/store/PipelineTriggersActionCreator';
+import DeleteOutlineIcon from '@material-ui/icons/DeleteOutline';
 import T from 'i18n-react';
-import { ISchedule, IGroupTrigger } from 'components/PipelineTriggers/store/ScheduleTypes';
+import { ISchedule, ICompositeTrigger } from 'components/PipelineTriggers/store/ScheduleTypes';
 import ConfirmationModal from 'components/shared/ConfirmationModal';
 import {
   PipelineName,
-  PipelineTriggerButton,
   StyledAccordion,
   StyledAccordionDetails,
   StyledAccordionSummary,
 } from 'components/PipelineTriggers/shared.styles';
 
 const TRIGGER_PREFIX = 'features.PipelineTriggers';
 
-const DisableGroupTriggerBtn = styled(PipelineTriggerButton)`
-  height: fit-content;
-  margin-top: 5px;
+const DisableCompositeTriggerBtn = styled.div`
```

Comment:
Buttons should be `Button`s for accessibility purposes

Reply:
updated